src/configuration/js-team/android.js

```diff
@@ -0,0 +1,43 @@
+import SETTINGS from '../perfherderSettings';
+
+const { frameworkIds } = SETTINGS;
+
+const AndroidConfig = {
+  label: 'Android',
+  benchmarks: {
+    speedometer: {
```

Comment:
Could you also please add unity?
As in here:
https://health.graphics/android
https://github.com/mozilla-frontend-infra/firefox-health-dashboard/blob/master/src/views/Android/index.jsx#L123-L140

<img width="606" alt="image" src="https://user-images.githubusercontent.com/44410/48074393-d215c000-e1ae-11e8-938a-6e6f2c7623ee.png">